service/controller/v5/arm_templates/ignition_container_setup.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "accessTier": {
+            "defaultValue": "Cool",
+            "type": "string"
+        },
+        "accountType": {
+            "defaultValue": "Standard_RAGRS",
+            "type": "string"
+        },
+        "blobContainer": {
+            "defaultValue": "ignition",
+            "type": "string"
+        },
```

Comment:
I think this should be more generic. Something like `container_setup.json` would make sense for me. `blobContainer` (which I would call `blobContainerName`) should come from the `key` package as it will be also used by one of the operator's resources to store the actual ignition.